### ISSUE TITLE
Integration into AutoPlug

### ISSUE THREAD

**Issue by Osiris-Team** [Skidamek/AutoModpack#248]
I wanted to support modpacks for a while now in autoplug, would you be interested in implementing this yourself? Of course if you got so free time at hands and find this enjoyable.

There should be an open issue related to this in the autoplug repo with more details.

**Comment by Skidamek** [comment]
Hey! I am not sure if you understand what AutoModpack actually does... 
I am pretty sure it works quite well with AutoPlug already.

AutoModpack doesn't install modpacks created on modrinth/curseforge. AutoModpack creates its own modpack from files on your server, so AutoPlug can nicely update whole modpack or whatever else just as is.

**Comment by Osiris-Team** [comment]
Yes you're right, its not the exact functionality your mod offers, but related. Basically your mod makes it easier for players to use modpacks, I wanted something similar inside autoplug, which benefits players and server owners too. For example a command to install and switch between modpacks easily. Even though the switching might even break things, on second thought.

**Comment by Skidamek** [comment]
Yeah, so give me exact list of things how would the integration look like because i have no idea...

**Comment by Osiris-Team** [comment]
https://github.com/Osiris-Team/AutoPlug-Client/issues/198
- ` .install mod pack <curseforge-mod-pack-id>` saves the current mod-pack and installs a new one.
- And a command to sync mods to the mc server version, to ensure everything is compatible.

**Comment by Skidamek** [comment]
Only thing i see worth to integrate would be that installation of modpack from curseforge. It instead of just install on server it could also place all modpack files inside the automodpack folders. It could be nice.

However i think its all to do on AutoPlug side, isn't it?

**Comment by Osiris-Team** [comment]
Yeah, however if you got existing code related to that (that doesn't depend on the fabric/forge) you can copy and paste it into autoplug I guess.

**Comment by Osiris-Team** [comment]
Additionally `.install mod pack <mod-pack-name>` would be nice to have to, and returning a list of similar mod-packs names to choose from, if multiple are available.

**Comment by Skidamek** [comment]
Yes sure.

Oh I just realized autoplug should also check for updates inside automodpack folders.

**Comment by Skidamek** [comment]
I don't have time/it's not my priority right now, but that all would be a nice feature someday. I would be up to it.

**Comment by Osiris-Team** [comment]
Yeah no worries.
Nice!
